pkg/machine/provider/platform.go

```diff
@@ -1,4 +1,4 @@
-//go:build (amd64 && !windows && amd64 && !darwin) || (arm64 && !windows && arm64 && !darwin) || (amd64 && darwin)
+//go:build (amd64 && !windows && amd64 && !darwin) || (arm64 && !windows && arm64 && !darwin)
```

Comment:
ok this is pre-existing but it looks overly complex?
Looks like it boils down to `!windows || !darwin` now so I recommend to change it.